bokeh/document.py

```diff
@@ -25,6 +26,38 @@
 
 DEFAULT_TITLE = "Bokeh Application"
 
+class DocumentProxy(object):
```

Comment:
UnlockedDocumentProxy or something like that might clarify a bit what this thing is